packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -52,6 +54,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
   private _dropDown: HTMLDivElement;
   private _dropdownLabel: HTMLElement;
   private _id: string;
+  private _multiSelect: boolean;
```

Comment:
I think we should not keep this internal variable. Otherwise you need to update it when props.multiSelect is changed.